R/bin.R

```diff
@@ -102,11 +105,11 @@ bin_breaks_width <- function(x_range, width = NULL, center = NULL,
 
 bin_breaks_bins <- function(x_range, bins = 30, center = NULL,
                             boundary = NULL, closed = c("right", "left")) {
-  if (length(x_range) != 2) abort("`x_range` must have two elements")
+  if (length(x_range) != 2) cli::cli_abort("{.arg x_range} must have two elements")
 
   bins <- as.integer(bins)
   if (bins < 1) {
-    abort("Need at least one bin.")
+    cli::cli_abort("Need at least one bin.")
```

Comment:
`bin` must be 1 or greater?

Reply:
I think `bins` can be derived from other arguments than `bin` so we shouldn't target that argument name directly

Reply:
It looks like this code path is only hit when `breaks` and `binwidth` are NULL, so I think it's only determined by `bins`.

Reply:
you are right - I'll update